### PR TITLE
Unit testing modifications

### PR DESCRIPTION
**Pull request by bobloblaws93**
Added unit tests for provider classes

### CODE REVIEW

File: src/providers/bank.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+
+import {environment} from "../environments/environment";
+import {BankTransferService} from './bank.service';
+import {AuthHttp} from "angular2-jwt";
+
+//check deposit
```
Comment (JTalarico) [inline src/providers/bank.service.spec.ts]: No in-function comments.

File: src/providers/bank.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+describe('Bank Service', () => {
+    beforeEach(() => {
+      this.BankTransferService = new BankTransferService();
+    });
+    //check if deposit is not null
```
Comment (JTalarico) [inline src/providers/bank.service.spec.ts]: Same. No in-function comments.

File: src/providers/currency.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import { AuthHttp } from "angular2-jwt";
+import {Observable} from "rxjs";
+import { TestBed, async, inject } from '@angular/core/testing';
+import { BaseRequestOptions, Http, HttpModule, Response, ResponseOptions } from '@angular/http';
+import { MockBackend } from '@angular/http/testing';
+import {Storage} from "@ionic/storage";
+
+import{CurrencyService} from "./currency.service";
+
+//currency service test
```
Comment (JTalarico) [inline src/providers/currency.service.spec.ts]: Use Javadoc style header.

/**
 *
**/

File: src/providers/wallet.service.spec.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+import { AuthHttp } from "angular2-jwt";
+import {Observable} from "rxjs";
+import {environment} from "../environments/environment";
+import{WalletsService} from "./wallet.service";
+
+
```
Comment (JTalarico) [inline src/providers/wallet.service.spec.ts]: Needs header describing the function.

**Review by JTalarico — CHANGES_REQUESTED** [review]
Please fix the changes regarding comment style.

**Comment by ghiobi** [comment]
@ronnpang1 Please make the changes and change the base. Pull request has been open for 7 days.

File: .project [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```
Comment (ghiobi) [inline .project:2]: Please remove file.

File: .settings/.jsdtscope [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.wst.jsdt.launching.JRE_CONTAINER"/>
```
Comment (ghiobi) [inline .settings/.jsdtscope]: Please remove file.

File: .settings/org.eclipse.wst.common.project.facet.core.xml [hunk 7]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<faceted-project>
+  <installed facet="webclipse.typescript" version="1.0"/>
```
Comment (ghiobi) [inline .settings/org.eclipse.wst.common.project.facet.core.xml]: Please remove file.

File: .settings/org.eclipse.wst.jsdt.ui.superType.container [hunk 8]
```diff
@@ -0,0 +1 @@
+org.eclipse.wst.jsdt.launching.JRE_CONTAINER
```
Comment (ghiobi) [inline .settings/org.eclipse.wst.jsdt.ui.superType.container]: Please remove file.

File: .settings/org.eclipse.wst.jsdt.ui.superType.name [hunk 9]
```diff
@@ -0,0 +1 @@
+Global
```
Comment (ghiobi) [inline .settings/org.eclipse.wst.jsdt.ui.superType.name]: Please remove file.

File: src/providers/auth.service.spec.ts [hunk 10]
```diff
@@ -0,0 +1,23 @@
+import { Storage } from '@ionic/storage';
+import { AuthHttp, tokenNotExpired } from 'angular2-jwt';
+import { Injectable, NgZone } from '@angular/core';
+
+import { environment } from '../environments/environment';
+import {AuthService} from './auth.service';
```
Comment (ghiobi) [inline src/providers/auth.service.spec.ts]: Please add space `{ AuthService }`

File: src/providers/auth.service.spec.ts [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+
+
+
+describe('Auth Service', () => {
+    beforeEach(() => {
```
Comment (ghiobi) [inline src/providers/auth.service.spec.ts]: Please fix the spaces.

File: src/providers/bank.service.spec.ts [hunk 12]
```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+
+import {environment} from "../environments/environment";
```
Comment (ghiobi) [inline src/providers/bank.service.spec.ts]: Please add spaces

File: src/providers/bank.service.spec.ts [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+import {BankTransferService} from './bank.service';
+import {AuthHttp} from "angular2-jwt";
+
+describe('Bank Service', () => {
+    beforeEach(() => {
+      this.BankTransferService = new BankTransferService();
+    });
```
Comment (ghiobi) [inline src/providers/bank.service.spec.ts]: Please fix the spacing.

File: src/providers/currency.service.spec.ts [hunk 14]
```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+import { AuthHttp } from "angular2-jwt";
+import {Observable} from "rxjs";
+import { TestBed, async, inject } from '@angular/core/testing';
+import { BaseRequestOptions, Http, HttpModule, Response, ResponseOptions } from '@angular/http';
+import { MockBackend } from '@angular/http/testing';
+import {Storage} from "@ionic/storage";
```
Comment (ghiobi) [inline src/providers/currency.service.spec.ts]: Please fix the spacing

File: src/providers/currency.service.spec.ts [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+
+import{CurrencyService} from "./currency.service";
+
+describe('Currency Service', () => {
+  
+    beforeEach(() => {
+         TestBed.configureTestingModule({
+
+              providers: [
+        {
```
Comment (ghiobi) [inline src/providers/currency.service.spec.ts]: please use double spaced indents

File: src/providers/wallet.service.spec.ts [hunk 16]
```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+import { AuthHttp } from "angular2-jwt";
+import {Observable} from "rxjs";
+import {environment} from "../environments/environment";
```
Comment (ghiobi) [inline src/providers/wallet.service.spec.ts]: Please add spaces .

File: src/providers/wallet.service.spec.ts [hunk 17]
```diff
@@ -0,0 +1,17 @@
...
+
+describe('Wallet Service', () => {
+    beforeEach(() => {
+       this.service = new WalletsService();
+    });
+     it('Get wallets response should not be null', () => {
+        expect(this.service.getWallets()).not.toBeNull();
+    });
+  
+    });
```
Comment (ghiobi) [inline src/providers/wallet.service.spec.ts]: Bad indentation

File: src/providers/wallet.service.spec.ts [hunk 18]
```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+import { AuthHttp } from "angular2-jwt";
+import {Observable} from "rxjs";
+import {environment} from "../environments/environment";
+import{WalletsService} from "./wallet.service";
+
+
+describe('Wallet Service', () => {
```
Comment (ghiobi) [inline src/providers/wallet.service.spec.ts]: Needs descriptions in comments

**Review by ghiobi — CHANGES_REQUESTED** [review]
Please make the changes.

File: src/providers/wallet.service.spec.ts [hunk 19]
```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+import { AuthHttp } from "angular2-jwt";
+import { Observable } from "rxjs";
+import { environment } from "../environments/environment";
+import{ WalletsService } from "./wallet.service";
+describe('Wallet Service', () => {
```
Comment (ghiobi) [inline src/providers/wallet.service.spec.ts:7]: Please add a space between imports and test.

File: src/providers/wallet.service.spec.ts [hunk 20]
```diff
@@ -0,0 +1,16 @@
...
+    beforeEach(() => {
+       this.service = new WalletsService();
+      });
```
Comment (ghiobi) [inline src/providers/wallet.service.spec.ts:10]: Weird indentation

File: src/providers/wallet.service.spec.ts [hunk 21]
```diff
@@ -0,0 +1,16 @@
...
+     
+     it('Get wallets response should not be null', () => {
+        expect(this.service.getWallets()).not.toBeNull();
+      });
+  
+    });
```
Comment (ghiobi) [inline src/providers/wallet.service.spec.ts:16]: Weird indentation

File: src/providers/currency.service.spec.ts [hunk 22]
```diff
@@ -0,0 +1,46 @@
...
+import { CurrencyService } from "./currency.service";
+/** 
+ * Currency test suite
+ */
+describe('Currency Service', () => {
+  
+    beforeEach(() => {
+         TestBed.configureTestingModule({
+              providers: [
+          {
```
Comment (ghiobi) [inline src/providers/currency.service.spec.ts:18]: Weird indentation

File: src/providers/currency.service.spec.ts [hunk 23]
```diff
@@ -0,0 +1,46 @@
...
+          provide: Http, useFactory: (backend, options) => {
+            return new Http(backend, options);
+          },
+          deps: [MockBackend, BaseRequestOptions]
+          },
+          Storage
+          MockBackend,
+          BaseRequestOptions,
+          CurrencyService
+      ]
```
Comment (ghiobi) [inline src/providers/currency.service.spec.ts:28]: Weird indentation

File: src/providers/bank.service.spec.ts [hunk 24]
```diff
@@ -0,0 +1,24 @@
...
+        expect(this.BankTransferService.deposit(2.00,'USD')).not.toBeNull();
+     });
+     
+     it('Withdrawl response should  be null', () => {
+        expect(this.BankTransferService.Withdrawl(2,'USD')).not.toBeNull();
+     });
+  
+    });
+
+  })
```
Comment (ghiobi) [inline src/providers/bank.service.spec.ts:24]: Weird indentation

File: src/providers/bank.service.spec.ts [hunk 25]
```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+import { environment } from "../environments/environment";
+import { BankTransferService } from './bank.service';
+import { AuthHttp } from "angular2-jwt";
+/** 
+ * Bank test suite
+ */
+describe('Bank Service', () => {
+    beforeEach(() => {
```
Comment (ghiobi) [inline src/providers/bank.service.spec.ts:10]: Weird indentation

Comment (ghiobi) [reply src/providers/bank.service.spec.ts:10]: Double space or tab?

**Review by ghiobi — CHANGES_REQUESTED** [review]
Please make the following changes.

**Pull request closed without merging**